src/Equations/acoustic/Kernels

```diff
@@ -0,0 +1 @@
+/dss/dsshome1/03/ge84lej2/fork-seissol/SeisSol/src/Equations/elastic/Kernels/
```

Comment:
Great to have it like that.

However, it'll need to be a relative symlink (i.e. something like `../../elastic/Kernels/`).